src/viewports/viewport.js

```diff
@@ -39,8 +54,29 @@ const DEFAULT_DISTANCE_SCALES = {
   degreesPerPixel: [1, 1, 1]
 };
 
+const DEFAULT_ZOOM = 0;
+
 const ERR_ARGUMENT = 'Illegal argument to Viewport';
 
+// Extracts or creates projection matrix from supplied options
+// Optionally creates a perspective or orthographic matrprojectionMatrix || ix
+function createPerspectiveProjectionMatrix({ // viewport arguments
```

Comment:
Should this be part of `math.gl`? Similar to [gl-matrix/mat4.perspective](http://glmatrix.net/docs/module-mat4.html#.perspective)

Reply:
> Should this be part of math.gl? Similar to gl-matrix/mat4.perspective

Yes, removed. This did more initially but ended up being essentially the same function as Matrix4().perspective.